docs/source/en/tutorials/custom_pipelines_components.md

```diff
@@ -0,0 +1,135 @@
+
+We allow loading of such pipelines by exposing a `trust_remote_code` argument inside [`DiffusionPipeline`]. The advantage of `trust_remote_code` lies in its flexibility. You can have different levels of customizations for a pipeline. Following are a few examples:
+
+* Only UNet is custom 
+* UNet and VAE both are custom
+* Pipeline is custom 
+* UNet, VAE, scheduler, and pipeline are custom 
+
+With `trust_remote_code=True`, you can achieve perform of the above!
+
+This tutorial covers how to author your pipeline repository so that it becomes compatible with `trust_remote_code`. You'll use a custom UNet, a custom scheduler, and a custom pipeline for this purpose. 
+
+<Tip warning={true}>
+
+You should use `trust_remote_code=True` _only_ when you fully trust the code and have verified its usage.
+
+</Tip>
+
+## Pipeline components
+
+In the interest of brevity, you'll use the custom UNet, scheduler, and pipeline classes that we've already authored:
+
+```bash
+# Custom UNet
+wget https://huggingface.co/sayakpaul/custom_pipeline_remote_code/raw/main/unet/my_unet_model.py
+# Custom scheduler
+wget https://huggingface.co/sayakpaul/custom_pipeline_remote_code/raw/main/scheduler/my_scheduler.py
+# Custom pipeline
+wget https://huggingface.co/sayakpaul/custom_pipeline_remote_code/raw/main/my_pipeline.py
+```
+
+<Tip warning={true}>
+
+The above classes are just for references. We encourage you to experiment with these classes for desired customizations.
+
+</Tip>
+
+Load the individual components, starting with the UNet:
+
+```python
+from my_unet_model import MyUNetModel
+
+pretrained_id = "hf-internal-testing/tiny-sdxl-custom-all"
+unet = MyUNetModel.from_pretrained(pretrained_id, subfolder="unet")
+```
+
+Then go for the scheduler:
+
+```python
+from my_scheduler import MyUNetModel
+
+scheduler = MyScheduler.from_pretrained(pretrained_id, subfolder="scheduler")
+```
+
+Finally, the VAE and the text encoders:
+
+```python
+from transformers import CLIPTextModel, CLIPTextModelWithProjection, CLIPTokenizer
+from diffusers import AutoencoderKL
+
+text_encoder = CLIPTextModel.from_pretrained(pretrained_id, subfolder="text_encoder")
+text_encoder_2 = CLIPTextModelWithProjection.from_pretrained(pretrained_id, subfolder="text_encoder_2")
+tokenizer = CLIPTokenizer.from_pretrained(pretrained_id, subfolder="tokenizer")
+tokenizer_2 = CLIPTokenizer.from_pretrained(pretrained_id, subfolder="tokenizer_2")
+
+vae = AutoencoderKL.from_pretrained(pretrained_id, subfolder="vae")
+```
+
+`MyUNetModel`, `MyScheduler`, and `MyPipeline` use blocks that are already supported by Diffusers. If you are using any custom blocks make sure to put them in the module files themselves. 
```

Comment:
I think it makes more sense to place this sentence after the code snippet with `MyPipeline` because it isn't introduced here yet

```suggestion
`MyUNetModel`, `MyScheduler`, and `MyPipeline` use blocks that are already supported by Diffusers. If you are using any custom blocks, make sure to put them in the module files themselves. 
```